cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -13,7 +12,6 @@
 pytestmark = [
     pytest.mark.long_running,
     pytest.mark.ignore_stream("upstream"),
-    pytest.mark.meta(blockers=[BZ(1677548, forced_streams=["5.11"])]),
```

Comment:
Can you add this BZ to the Bugzilla doc as a reference for the test cases which were affected by it?

Reply:
@valaparthvi I have added this Tracker BZ, to monitor/verifyAnsible related BZ. Hence not adding this BZ in each test cases for reference.